doc/userguide/performance/runmodes.rst

```diff
@@ -46,3 +46,21 @@ useful during development.
 
 For more information about the command line options concerning the
 runmode, see :doc:`../command-line-options`.
+
+Load balancing
+~~~~~~~~~~~~~~
+
+Suricata may use different ways to load balance the packets to process
+between different threads with the configuration option `autofp-scheduler`.
+
+The default value is `hash`, which means the packet is assigned to threads
+using the 5-7 tuple hash, which is also used anyways to store the flows
+in memory.
+
+This option can also be set to
+- `ippair` : packets are assigned to threads using addresses only.
+- `ftp-hash` : same as `hash` except for flows that may be ftp or ftp-data
```

Comment:
Could this be true for any protocol that has a control flow and a data flow? SIP comes to mind, though I don't think we do anything with the data path there (yet). I wonder if a better name is `related` (taken from iptables terminology)?

Any reason this is not done by default?

Reply:
Who can test ?

And is this the right place to implement such a change ?